applications/GeoMechanicsApplication/MainKratos_multiple_stages_template.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+import os
+import KratosMultiphysics
+import KratosMultiphysics.ExternalSolversApplication
+# import KratosMultiphysics.TrilinosApplication as TrilinosApplication
+import KratosMultiphysics.StructuralMechanicsApplication
+import KratosMultiphysics.GeoMechanicsApplication
```

Comment:
```suggestion
```

I strongly recommend only importing what you need, this is how we do it for the main apps